llvmlite/tests/test_binding.py

```diff
@@ -3075,6 +3141,36 @@ def test_run(self):
         self.assertIn("%.4", orig_asm)
         self.assertNotIn("%.4", optimized_asm)
 
+    def test_run_O0(self):
+        pb = self.pb(0)
+        mod = self.module()
+        fun = mod.get_function("sum")
+        orig_asm = str(fun)
+        fpm = pb.getFunctionPassManager()
+        fpm.run(fun, pb)
+        optimized_asm = str(fun)
```

Comment:
(following from above comment)
```suggestion
        orig_asm, optimized_asm = self.run_o_n(0)
```

... so that the difference between the O3 and O0 runs is a little easier to spot on inspection (either the presence or absence of `"%.4"` in the optimized assembly)